### PR TITLE
#1160 add styling to popular tags

### PR DESCRIPTION
**Pull request by cweave**
#1160 add styling to popular tags

Light mode, python hover/active state:
image

Dark mode, python hove/active state:
image

iPhone 12/13 screensize (375px wide):
image

- This change updates the `popular-tag` list to a `div` with links. Adds `titles` and tabbing order to the links for a11y. 
- Adjusted color variables for better control over opacity. Converted `container-border` color to rbg value.
- Added `container-border-alt` color for darkmode

### CODE REVIEW

File: _includes/scripts.html [hunk 1]
```diff
@@ -46,11 +46,11 @@
   </div>
   <div class="filters-panel cf">
     <label class="filter-label" >Popular tags:</label>
-    <ul class="popular-tags">
+    <div class="popular-tags">
     <% _.each(popularTags, function(entry, key){ %>
-      <li><a><%-entry.name%></a> (<%-entry.frequency%>)</li>
+      <a title="Popular Tag: <%-entry.name%>" tabindex="0"><%-entry.name%> <span class="popular-tags__frequency">(<%-entry.frequency%>)</span></a>
```
Comment (shiftkey) [inline _includes/scripts.html]: 🤔 Can we do these styling changes without losing the `ul`/`li` semantics of these elements?

**Review by shiftkey — COMMENTED** [review]
I like where this is heading - my only question is about the DOM changes that support this styling...

**Comment by AbsolytnaiaLove** [comment]
сб, 8 жовт. 2022, 17:41 користувач Brendan Forster ***@***.***>
пише:

**Comment by AbsolytnaiaLove** [comment]
сб, 8 жовт. 2022, 23:21 користувач AbsolytnaiaLove ***@***.***>
пише:

File: _includes/scripts.html (hunk 1 above)
Comment (cweave) [reply _includes/scripts.html]: @shiftkey sure thing! see this commit here #[e902028](https://github.com/up-for-grabs/up-for-grabs.net/pull/3460/commits/e9020282dce70f7e6fe1d2d7d136c7305bd62e14)

File: javascripts/main.js [hunk 2]
```diff
@@ -154,7 +154,7 @@ define([
       });
 
     projectsPanel
-      .find('ul.popular-tags')
+      .find('div.popular-tags')
```
Comment (shiftkey) [inline javascripts/main.js]: If I vitsit https://up-for-grabs.net/# and click on the `javascript` tag I see the URL change to `https://up-for-grabs.net/#/filters?tags=javascript` and the form shows that `javascript` is in the list:



However if I visit the staging site for this PR https://deploy-preview-3460--up-for-grabs-test-bench.netlify.app/#/ and click the `javascript` nothing happens. 

I think this needs to change back to `ul` as this appears to be the click handlers for the tags:

```suggestion
      .find('ul.popular-tags')
```

**Review by shiftkey — CHANGES_REQUESTED** [review]

File: javascripts/main.js (hunk 2 above)
Comment (cweave) [reply javascripts/main.js]: ooof that's embarrassing 🤦‍♀️. sorry about that, should be all set now

**Comment by ritwik12** [comment]
@cweave Thanks

**Pull request merged**